### DISCUSSION TITLE
Is there a plan to develop a version for OpenHarmony?

### DISCUSSION THREAD

**Discussion by fukyou-fukme** [ruffle-rs/ruffle#18903] [Ideas]
Is there a plan to develop a version for OpenHarmony?

**Comment by Lord-McSweeney** [comment]
Ruffle should work on any system with Rust and OpenGL support, though it probably won't run well on a smaller embedded system.

**Reply by minusmulticoloredper** [reply]
Currently, there is a closed-source distribution in China based on OpenHarmony called HarmonyOS Next.
Smartphones equipped with HarmonyOS Next have already appeared in the Chinese market.
HarmonyOS Next supports OpenGL, and it should support any cross-platform libraries.
